DOCAPI-7438: Update of quantileTiming docs. (#6455)
ClickHouse/ClickHouse: docs/en/query_language/agg_functions/reference.md

Don't use this function for calculating timings. There is a more suitable function for this purpose: `quantileTiming`.

## quantileTiming {#agg_function-quantiletiming}

Computes the quantile of the specified level with determined precision. The function intended for calculating quantiles of page loading time in milliseconds. 

```
quantileTiming(level)(expr)
```

**Accuracy**

The calculation is accurate if:

- Total number of values is not more than about 5670.
- Total number of values is more than about 5670, but the times of page loading is less than 1024ms.

Otherwise, the result of a calculation is rounded to the value, multiple of 16 ms.

> For calculating quantiles of page loading times, using this function is more effective, and the result is more accurate than for the [quantile](#agg_function-quantile) function.

Reviewer: ```suggestion
!! note "Note"
    For calculating quantiles of page loading times, this function is more effective and accurate compared to [quantile](#agg_function-quantile).
```

> **Parameters**

Reviewer: I'd swap Parameters and Accuracy

- `level` — Quantile level. Range: [0, 1].

> - `expr` — [Expression](../syntax.md#syntax-expressions) returning number in the [Float*](../../data_types/float.md) type.

Reviewer: we'd better explicitly mention that it's supposed to be in milliseconds (btw why it's float then?)

Author: From the dev pull: "Returned back support for floating point argument in function quantileTiming for backward compatibility."

>     - When passing negative values to the function, the behavior is undefined.

Reviewer: ```suggestion
    - If negative values are passed to the function, the behavior is undefined.
```

    - If the value is greater than 30,000 (a page loading time of more than 30 seconds), it is assumed to be 30,000.

**Returned value**

- Quantile of the specified level.

Type: `Float32`.

> If no values were passed to the function (when using `quantileTimingIf`), [NaN](../../data_types/float.md#data_type-float-nan-inf) is returned. The purpose of this is to differentiate these cases from the cases which result in zero. See [ORDER BY clause](../select.md#select-order-by) for the note on sorting `NaN` values.

Reviewer: this deservers note/warning block

> The result is determinate (it doesn't depend on the order of query processing).

Reviewer: ```suggestion
The result is deterministic (it doesn't depend on the order of query processing).
```
